clients/src/main/java/org/apache/kafka/clients/producer/Partitioner.java

```diff
@@ -31,13 +34,27 @@ public interface Partitioner extends Configurable, Closeable {
      *
      * @param topic The topic name
      * @param key The key to partition on (or null if no key)
-     * @param keyBytes The serialized key to partition on( or null if no key)
+     * @param keyBytes The serialized key to partition on(or null if no key)
```

Comment:
nit: Can we change:

`The serialized key to partition on(or null if no key)`

to

`The serialized key to partition on (or null if no key)`

Reply:
I have formatted the documentation.